templates/frontend/pages/indexJournal.tpl

```diff
@@ -23,18 +23,37 @@
 {include file="frontend/components/header.tpl" pageTitleTranslated=$currentJournal->getLocalizedName()}
 
 <main class="container main__content" id="main">
-	{if $journalDescription}
-		<section class="journal-desc">
-			<h2 class="journal-desc__title">About this journal</h2>
-			<div class="row">
-				<div class="col-sm-8 journal-desc__content">
-					{$journalDescription|strip_unsafe_html}
-				</div>
-			</div>
-			{capture assign="aboutPageUrl"}{url router=$smarty.const.ROUTE_PAGE page="about"}{/capture}
-			<p><a href="{$aboutPageUrl}" class="btn btn-primary">{translate key="plugins.themes.pragma.more-info"}</a>
-			</p>
-		</section>
+	{if $journalDescription or $announcements}
+	<header class="row index-header">
+		{if $journalDescription}
+			<section class="col-sm-6 journal-desc">
+				<h2 class="issue__meta">{$displayPageHeaderTitle}</h2>
+				<article>
+					<h3 class="journal-desc__title">{translate key="about.aboutContext"}</h3>
+					{$journalDescription|strip_unsafe_html|truncate:450}
+					<p>
+						{capture assign="aboutPageUrl"}{url router=$smarty.const.ROUTE_PAGE page="about"}{/capture}
+						<a href="{$aboutPageUrl}" class="btn btn-primary">{translate key="plugins.themes.pragma.more-info"}</a>
+					</p>
+				</article>
+			</section>
+		{/if}
+		{if $announcements}
```

Comment:
@sssoz, here and on line 26 we need to perform additional checks.
1) $announcements here is an array, so need to be checked if it's empty, e.g., $announcements|@count
2) Also, a user can choose if actually announcements should be displayed on the index page and their number; $numAnnouncementsHomepage can tell us if the number is set and it's greater than 0. 
3) To be reusable this check can be assigned to an internal smarty variable: {if ...}{assign var="showAnnouncements" value=true}{/if}

Reply:
Thanks! will fix it now

Reply:
Fixed, and adjusted for cases in which users would like to have more than one latest announcement by displaying it in a Bootstrap carousel (it works for any number of announcements shown on the homepage).

Force-pushed this!